File: SchedentFrontend/src/app/models/schedule/scheduleModel.ts
export class ScheduleModel {
    public scheduleType: string;
    public subject: string;
    public professor: string;
    public subgroup: string;
    public week: number;
    public day: string;
    public duration: number;
    public startsAt: string;

    public constructor(scheduleType: string, subject: string, professor: string, subgroup: string, week: number, day: string, duration: number, startsAt: string){
        this.scheduleType = scheduleType;
        this.subject = subject;
        this.professor = professor;
        this.subgroup = subgroup;
        this.week = week;
        this.day = day;
        this.duration = duration;
        this.startsAt = startsAt;
    }
}

File: SchedentAPI/Schedent.Domain/DTO/Schedule/ScheduleModel.cs
namespace Schedent.Domain.DTO.Schedule
{
    public class ScheduleModel
    {
        public string ScheduleType { get; set; }

        public string Subject { get; set; }

        public string Professor { get; set; }

        public string Subgroup { get; set; }

        public int Week { get; set; }

        public float Duration { get; set; }

        public string StartsAt { get; set; }
    }
}
